java/src/Ice/src/main/java/com/zeroc/Ice/UnknownException.java

```diff
@@ -45,7 +27,7 @@ public String ice_id() {
   }
 
   /** This field is set to the textual representation of the unknown exception if available. */
-  public String unknown;
+  public final String unknown;
```

Comment:
The unknown field is replaced by the exception message.

Reply:
I removed the field.